icdiff

```diff
@@ -57,6 +56,16 @@ color_mapping = {
 }
 
 
+def _split(s, sep):
+    return list(map(str.strip, s.split(sep)))
+
+
+color_str = os.getenv('ICDIFF_COLORS')
+if color_str:
+    env_map = dict(tuple(_split(s, ':')) for s in _split(color_str, ','))
```

Comment:
I think this would be more pythonic as:

    dict(tuple(x.strip() for x in s.split(':'))
         for s in color_str.split(','))